### PR TITLE
java: parse_jvm_version(): Handle single-digit build number suffix

### PR DESCRIPTION
**Pull request by Jongy**
For example - "1.8.0_292-b2"

Also fixes
```
'openjdk version "1.8.0_212-ojdkbuild"\nOpenJDK Runtime Environment (build 1.8.0_212-ojdkbuild-04)\nOpenJDK 64-Bit Server VM (build 25.212-b04, mixed mode)\n'
```

### CODE REVIEW

**Comment by Jongy** [comment]
I'll re-run the query within a few days to get more unique samples, and if it passes, we'll merge.

**Comment by Jongy** [comment]
> I'll re-run the query within a few days to get more unique samples, and if it passes, we'll merge.

I re-ran, it's okay

File: gprofiler/profilers/java.py [hunk 1]
```diff
@@ -557,8 +557,8 @@ def parse_jvm_version(version_string: str) -> JvmVersion:
     assert m is not None, f"did not find build_str in {version_string!r}"
     build_str = m.group(1)
 
-    if version_str.endswith("-internal") or version_str.endswith("-ea"):
-        # strip the "internal" or "early access" suffixes
+    if version_str.endswith("-internal") or version_str.endswith("-ea") or version_str.endswith("-ojdkbuild"):
```
Comment (michelhe) [inline gprofiler/profilers/java.py:560]:
```suggestion
    if any(version_str.endswith(suffix) for suffix in ("-internal", "-ea", "-ojdkbuild")):
```

**Review by michelhe — CHANGES_REQUESTED** [review]
.

**Review by michelhe — APPROVED** [review]

**Pull request merged**